_layouts/news-item.html

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
```

Comment:
```suggestion
layout: deep-page
```

i think this would technically be a deep page and should probably have breadcrumbs

Reply:
the thing about having a deep-page approach to this is that it wouldn't put the headline inside the article tag (from what i imagine), so maybe this should be something brand new..